libs/accounts/recovery-phone/src/lib/recovery-phone.service.ts

```diff
@@ -186,6 +186,12 @@ export class RecoveryPhoneService {
       uid
     );
 
+    // TBD: Random, obs. why do we do this? It seems like a potential edge case, what if the user
```

Comment:
I think the main rationale was that we didn't want a user to have 2FA enabled with no backup recovery method.